C/CUDA/CUDA@10.2/build_tarballs.jl

```diff
@@ -23,36 +23,44 @@ rm -rf ${prefix}/include/thrust
 
 # binaries
 mkdir -p ${bindir} ${libdir} ${prefix}/lib ${prefix}/share
-if [[ ${target} == x86_64-linux-gnu ]]; then
+if [[ ${target} == x86_64-linux-gnu || ${target} == aarch64-linux-gnu ]]; then
+    if [[ ${target} == x86_64-linux-gnu ]]; then
```

Comment:
I'd rather make the CUDA_full_jll move everything in the correct spot so that we only have to special case in a single build script.

Reply:
Completely agree.

Reply:
`lib64` should be a symlink to the target-specific directory.

Reply:
```
tim@taurus ~$ ls -la /opt/cuda                                                                               
total 156K
drwxr-xr-x 1 root root 252 feb 11 13:09 ./
drwxr-xr-x 1 root root 364 feb 11 17:21 ../
drwxr-xr-x 1 root root 354 feb 11 13:09 bin/
drwxr-xr-x 1 root root 134 feb  7 08:31 extras/
lrwxrwxrwx 1 root root  28 jan 21 00:12 include -> targets/x86_64-linux/include/
drwxr-xr-x 1 root root  74 feb  7 08:31 integration/
lrwxrwxrwx 1 root root  24 jan 21 00:12 lib64 -> targets/x86_64-linux/lib/
drwxr-xr-x 1 root root  14 sep  6 15:25 nvml/
drwxr-xr-x 1 root root  78 sep  6 15:25 nvvm/
drwxr-xr-x 1 root root   6 sep  6 15:25 share/
drwxr-xr-x 1 root root 262 feb  7 08:31 src/
drwxr-xr-x 1 root root  24 sep  6 15:25 targets/
drwxr-xr-x 1 root root  58 feb  7 08:31 tools/
-rw-r--r-- 1 root root  25 jan 21 00:12 cublas_version.txt
-rw-r--r-- 1 root root 70K jan 21 00:12 CUDA_Toolkit_Release_Notes.txt
-rw-r--r-- 1 root root 160 jan 21 00:12 DOCS
-rw-r--r-- 1 root root 61K jan 21 00:12 EULA.txt
-rw-r--r-- 1 root root 524 jan 21 00:12 README
```